admin/services/marking.service.js

```diff
@@ -43,23 +46,32 @@ markingService.mark = async function (completedCheck) {
     processedAt: moment.utc()
   }
 
+  const formData = JSON.parse(checkForm.formData)
+
   // Store the mark for each answer
   const answers = []
   let questionNumber = 1
 
-  for (let answer of completedCheck.data.answers) {
-    const data = R.clone(answer)
-    data.answer = R.slice(0, 60, answer.answer)
-    data.questionNumber = questionNumber
+  for (let question of formData) {
+    const currentIndex = questionNumber - 1
+    const answer = completedCheck.data.answers[currentIndex]
+    const currentAnswer = answer && answer.answer
+    let data = R.clone(answer)
```

Comment:
It doesn't seem particularly clear to me that `data` has properties `factor1`, `factor2`, `answer`, `questionNumber`.